### ISSUE TITLE
Is elastalert compatible with Elasticsearch 6.2?

### ISSUE THREAD

**Issue by Amruth007** [Yelp/elastalert#1645]
Hi,

I have elasticsearch 5.6.7 and I am not using elastalert. Now I am planning to upgrade elasticsearch to 6.2 and then use elastalert only after it's upgraded to 6.2. Will it be possible to use elastalert with elasticsearch 6.2?

Thanks

**Comment by sean-kang** [comment]
I use ElastAlert with AWS Elasticsearch 6.2. It works okay for me.

**Comment by Amruth007** [comment]
Hi Sean,

You say that it works with clean installation of elasticsearch 6.2 without prior indexes created in older versions of elasticsearch? Someone mentioned that it's not possible to use elastalert on clean installation of 6.2. And also I am not using elasticsearch 6.2 on AWS. Please shed some light on this issue.

**Comment by kskewes** [comment]
I also used ElastAlert with clean 6.2 install on Kubernetes using Clockworksoul? helm chart. I think I used a different 3rd party ElastAlert dockerfile though. 
Be great if Yelp merged the PR #1644 and ran at least an 'official' docker image. Helm chart nice to have, just image be good start.
